jenkins/prepareOskar.ps1

```diff
@@ -1,4 +1,19 @@
-﻿$HDD = $(Split-Path -Qualifier $env:WORKSPACE)
+﻿Function proc($process,$argument)
+{
+    $p = Start-Process $process -ArgumentList $argument -NoNewWindow -PassThru
+    $h = $p.Handle
```

Comment:
Why do we need this handle?

Reply:
To fix the broken Powershell -wait parameter. Don't ask me why it works, it does. Also it is just a copy of the Funktion in the main helper.ps1